### DISCUSSION TITLE
My automation to save battery

### DISCUSSION THREAD

**Discussion by sebatze** [Ludy87/xplora_watch#104] [Show and tell]
I noticed that the xplora battery drains pretty quickly, if the status is polled every 180 seconds.
Therefore, I made this automation, maybe that is useful for someone else.

**Prerequisits:**

- xplora watch is configured to use your home wifi
- Device tracker based on wifi (I use https://www.home-assistant.io/integrations/fritz)

**How to**

- In the xplora config, set the scan interval to a higher number, e.g. 3600s (1 hour)
- create an automation, that runs every x minutes, where x is your preferred interval.
- Conditions for the automation: 
 - Wifi-Device-Tracker for the watch shows "not-home" 
 - Battery is above 30% (or whatever number you like)
 - Optional: Family.group is home (I don't want home assistant to track the watch if nobody is home)
- Action: call the service xplora_watch.see

**Comment by luciandf** [comment]
i did notice a faster battery drain since I configured this integration. so the main reason appears to be the scan interval?

I cannot use an automation like yours because my homeassistant server is located elsewhere but basically your automation forces an update of the sensor data at specified intervals?

**Reply by sebatze** [reply]
yes, that is correct. 
For example you could update less often during school or at night...

**Reply by luciandf** [reply]
and to update it less often, I have to use an automation like you did. is that correct?

**Comment by luciandf** [comment]
> * Wifi-Device-Tracker

Do you mind posting an example for your automation?

**Comment by sebatze** [comment]
I don't use this anymore, kids grew out of using xplora.


However, here is my automation:

alias: Xplora Device Tracking Ein- und ausschalten
description: ""
mode: single
triggers:
  - entity_id:
      - group.family
    from: home
    for:
      hours: 0
      minutes: 5
      seconds: 0
    id: ausschalten
    trigger: state
  - entity_id: sensor.Kind_watch_battery
    below: "30"
    id: ausschalten
    trigger: numeric_state
  - entity_id:
      - device_tracker.Kind_watch_tracker_xxx
    for:
      hours: 0
      minutes: 1
      seconds: 0
    from: null
    to: home
    id: ausschalten
    trigger: state
  - value_template: >-
      {{ states('device_tracker.Kind_watch_tracker_xxx') ==
      "schule"}}
    for:
      hours: 0
      minutes: 5
      seconds: 0
    id: ausschalten
    alias: Kind bei schule
    enabled: false
    trigger: template
  - entity_id:
      - device_tracker.Kind_watch_tracker_xxx
    for:
      hours: 0
      minutes: 1
      seconds: 0
    from: null
    to: not_home
    id: einschalten
    trigger: state
  - at: input_datetime.schulende_Kind_heute
    id: Schulende
    trigger: time
  - at: "14:30:00"
    id: Schulende
    alias: 14:00 an Schultagen - etwas vor Abholung um 15:00
    trigger: time
  - entity_id:
      - device_tracker.Kind_watch_tracker_xxx
    to: schule
    id: ausschalten
    trigger: state
conditions: []
actions:
  - if:
      - condition: trigger
        id:
          - ausschalten
    then:
      - data: {}
        target:
          entity_id: input_boolean.xplora_tracker_switch
        action: input_boolean.turn_off
    else: []
  - if:
      - condition: or
        conditions:
          - condition: trigger
            id:
              - einschalten
          - condition: and
            conditions:
              - condition: trigger
                id:
                  - Schulende
              - condition: state
                entity_id: binary_sensor.workday_sensor
                state: "on"
              - condition: state
                entity_id: calendar.ferien_nrw
                state: "off"
              - condition: template
                value_template: >-
                  {{ states('device_tracker.Kind_watch_tracker_xxx') ==
                  "schule"}}
    then:
      - data: {}
        target:
          entity_id: input_boolean.xplora_tracker_switch
        action: input_boolean.turn_on

**Reply by luciandf** [reply]
thank you!
